Source/WebCore/svg/SVGAnimateMotionElement.cpp

```diff
@@ -185,19 +185,6 @@ bool SVGAnimateMotionElement::setToAtEndOfDurationValue(const String& toAtEndOfD
     return true;
 }
 
-void SVGAnimateMotionElement::buildTransformForProgress(AffineTransform* transform, float percentage)
```

Comment:
Because you deleted this function, the code is duplicated below. Please keep this method and a new parameter `unsigned repeatCount` to it and make it return `PathTraversalState`.

Reply:
Alternatively you can split this into two functions:
```
static PathTraversalState traversalStateAtPositionOnPath(const Path& path, float percentage)
{

}

static void buildTransformForProgress(AffineTransform* transform, const PathTraversalState& traversalState, unsigned repeatCount)
{

}
```